src/library/searchqueryparser.cpp

```diff
@@ -263,6 +265,28 @@ void SearchQueryParser::parseTokens(QStringList tokens,
     }
 }
 
+std::unique_ptr<AndNode> SearchQueryParser::parseAndNode(const QString& query) const {
+    auto pQuery(std::make_unique<AndNode>());
```

Comment:
we prefer here the = initializer
```suggestion
    auto pQuery = std::make_unique<AndNode>();
```